core/jvm/src/test/scala/scalaz/zio/RTSSpec.scala

```diff
@@ -472,6 +474,24 @@ class RTSSpec(implicit ee: ExecutionEnv) extends AbstractRTSSpec with AroundTime
     unsafeRun(io) must_=== 42
   }
 
+  def testBracketUseIsInterruptible = {
+    val io =
+      for {
+        fiber <- IO.bracket[Nothing, Unit, Unit](IO.unit)(_ => IO.never)(_ => IO.unit).fork
```

Comment:
The second parameter group is for `release`, the third for `use`, shouldn't this be:
```scala
fiber <- IO.bracket[Nothing, Unit, Unit](IO.unit)(_ => IO.unit)(_ => IO.never).fork
```
instead?

Just asking because the test passes in that case 😅

Reply:
Oh 😅 
Bracket0 test still fails with correct order though!

Reply:
Yes, I'm on it.